recipes/libsecret/all/conanfile.py

```diff
@@ -0,0 +1,86 @@
+from conans import ConanFile, Meson, tools
+from conans.errors import ConanInvalidConfiguration
+import os
+
+
+class LibsecretConan(ConanFile):
+    name = "libsecret"
+    description = "A library for storing and retrieving passwords and other secrets"
+    topics = ("conan", "libsecret", "gobject", "password", "secret")
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://wiki.gnome.org/Projects/Libsecret"
+    license = "LGPL-2.1"
+    generators = "pkg_config"
+
+    settings = "os", "arch", "compiler", "build_type"
+    options = {"shared": [True, False], "fPIC": [True, False]}
+    default_options = {"shared": False, "fPIC": True}
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    @property
+    def _build_subfolder(self):
+        return "build_subfolder"
+
+    def validate(self):
+        if self.settings.os == "Windows":
+            raise ConanInvalidConfiguration(
+                "libsecret recipe is not yet compatible with Windows."
+            )
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
```

Comment:
do you need to handle windows if windows isn't supported anyway?

Reply:
The conan hooks demand it. See failure above, before I added the last commit.

Reply:
True, I'll push a PR fixing it on hooks.

Reply:
Great! Should I wait for the fix and remove this then?